docs/user/aws/limits.md

```diff
@@ -0,0 +1,74 @@
+# AWS Limits
+
+You can find a comprehensive list of the default AWS service limits published here:
+
+[AWS Service Limits][service-limits]
+
+Below, we’ll identify OpenShift cluster needs and how those impact some of those limits.
+
+## S3
+
+There is a limit of 100 S3 buckets per account. The installation creates a bucket temporarily. Also, the registry 
+component creates a permanent bucket. This will limit the number of clusters per account to 99.
+
+## VPC
+
+Each cluster creates its own VPC. The default limit of VPCs per region is 5 and will allow 5 clusters. To have more 
+than 5 clusters, you will need to increase this limit.
+
+## Elastic Network Interfaces (ENI)
+
+The default installation creates 21 + the number of availability zones of ENIs (e.g. us-east-1 = 21 + 6 = 27 ENIs). 
```

Comment:
Is this true?  What are the 21 for?  Three masters with public IPs, and...?

Reply:
It is. They may be occupied by the ALB targets, but they are assigned to ELBs. Also, one per instance is attached. I don't know why, private IPs assigned in the subnet via DHCP are permanent for the lifetime of the node (regardless of stops/starts). Usually, these are created so the same IP can be given/reassigned to another instance; I don't think this is a problem we have.